chucks-pick3-web/angular-app/src/app/components/customer-summary.component/customer-summary.component.ts

```diff
@@ -45,6 +38,14 @@ export class CustomerSummaryComponent implements OnInit {
     }
   }
 
+  public selectCustomerData(targetCustomer) {
```

Comment:
Do we even need the `selectCustomerData(customer)` method?

Reply:
Yes, this method pulls the information from the main array and displays it in the next component in order to be viewed and edited.

Reply:
Resolved requested changes.